ResourceStatusSystem/Utilities/InfoGetter.py

```diff
@@ -45,85 +45,86 @@ def getPoliciesThatApply( decisionParams ):
   #   ...
 
   # Get policies that match the given decisionParameters
-  for policyName, policySetup in policiesConfig.items():
+  for policyName, policySetup in policiesConfig.iteritems():
 
     # The parameter policyType replaces policyName, so if it is not present,
     # we pick policyName
     try:
-      policyType = policySetup[ 'policyType' ][ 0 ]
+      policyType = policySetup['policyType'][0]
     except KeyError:
       policyType = policyName
-      #continue
+      # continue
 
     # The section matchParams is not mandatory, so we set {} as default.
-    policyMatchParams  = policySetup.get( 'matchParams',  {} )
-    gLogger.debug("matchParams of %s: %s" %(policyName, str(policyMatchParams)))
+    policyMatchParams = policySetup.get('matchParams', {})
+    gLogger.debug("matchParams of %s: %s" % (policyName, str(policyMatchParams)))
 
     # FIXME: make sure the values in the policyConfigParams dictionary are typed !!
     policyConfigParams = {}
-    #policyConfigParams = policySetup.get( 'configParams', {} )
-    policyMatch = Utils.configMatch( decisionParams, policyMatchParams )
-    gLogger.debug("PolicyMatch for decisionParams %s: %s" %(decisionParams, str(policyMatch)))
-    policyFilter = _filterPolicies( decisionParams, policyMatchParams )
+    # policyConfigParams = policySetup.get( 'configParams', {} )
+    policyMatch = Utils.configMatch(decisionParams, policyMatchParams)
+    gLogger.debug("PolicyMatch for decisionParams %s: %s" % (decisionParams, str(policyMatch)))
 
-    #WARNING: we need an additional filtering function when the matching
-    #is not straightforward (e.g. when the policy specify a 'domain', while
-    #the decisionParams has only the name of the element)
-    if policyMatch and policyFilter:
-      policiesThatApply.append( ( policyName, policyType, policyConfigParams ) )
+    # WARNING: we need an additional filtering function when the matching
+    # is not straightforward (e.g. when the policy specify a 'domain', while
+    # the decisionParams has only the name of the element)
+    if policyMatch and _filterPolicies(decisionParams, policyMatchParams):
+      policiesThatApply.append((policyName, policyType, policyConfigParams))
 
-  gLogger.debug("policies that apply: %s" %str(policiesThatApply))
+  gLogger.debug("policies that apply (before post-processing): %s" % str(policiesThatApply))
+  policiesThatApply = postProcessingPolicyList(policiesThatApply)
+  gLogger.debug("policies that apply (after post-processing): %s" % str(policiesThatApply))
```

Comment:
I only think of this because I see both in this PR, but what's the policy between `verbose` and `debug` ?

Reply:
My understanding on this is that `verbose` is what you may want to see in a component log, and `debug` is purely for what you want to... debug.

Reply:
Agree with @fstagni . Verbose is to monitor "normal" system behavior. debug is for understanding problems.